packages/themes/src/components/ThemeStyles.vue

```diff
@@ -1,12 +1,20 @@
 <script lang="ts" setup>
 import '../base.css'
 import { type ThemeId, getThemeById } from '../index'
+import DefaultFonts from './DefaultFonts.vue'
 
-defineProps<{
-  id?: ThemeId
-}>()
+withDefaults(
+  defineProps<{
+    id?: ThemeId
+    withoutDefaultFonts?: boolean
+  }>(),
+  {
+    withoutDefaultFonts: false,
+  },
+)
 </script>
 <template>
+  <DefaultFonts v-if="!withoutDefaultFonts" />
```

Comment:
Does this work? I was under the impression CSS imports always get imported if the component is imported anywhere regardless of whether or not it's rendered.

Reply:
Works locally, but could be true for the build. I’ll check that.

Reply:
Really good find, it doesn’t work. :)